### PR TITLE
Upload factory and inventory tutorial examples

### PR DESCRIPTION
**Pull request by ejunprung**
Still verifying that these examples work fine

### CODE REVIEW

File: tests/examples/inventory_management/inventory_env_pathmind.py [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+            low=np.zeros(m-1), high=self.supply_capacity, dtype=np.int16)
+        # observation space (Inventory position at each echelon, which is any integer value)
+        self.observation_space = gym.spaces.Box(
+            low=-np.ones(self.pipeline_length)*self.supply_capacity.max()*self.num_periods*10,
+            high=np.ones(self.pipeline_length)*self.supply_capacity.max()*self.num_periods, dtype=np.int32)
+
+    def number_of_agents(self) -> int:
+        return 1
+
+    # def action_space(self, agent_id) -> typing.Union[Continuous, Discrete]:
```
Comment (ejunprung) [inline tests/examples/inventory_management/inventory_env_pathmind.py:169]: @maxpumperla I'm trying to convert the OR Gym inventory model to a Pathmind Simulation but I'm confused about the action space setup. I think it's continuous, how should that be set up in this case?

File: tests/examples/factory/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+pathmind==0.3
+ruamel.yaml
```
Comment (slinlee) [inline tests/examples/factory/requirements.txt:2]: Try this. the installed ruamel.yaml is an old version so let's install a new one.

```suggestion
ruamel.yaml>=0.17
```

File: tests/examples/inventory_management/inventory_env_pathmind.py (hunk 1 above)
Comment (maxpumperla) [reply tests/examples/inventory_management/inventory_env_pathmind.py:169]: Union of Countinuous and Discrete tells you that the return type is either one of them. If you have a Continuous space, you create and return that. If it's discrete, you use discrete.

Comment (ejunprung) [reply tests/examples/inventory_management/inventory_env_pathmind.py:169]: Sorry, I meant to ask if you understand the action space in this particular simulation. 

I read code like a 90-year old so it takes me a long time. I was hoping you would know at a glance to save me a couple of hours. I was just being selfish so no worries if you don't have the bandwidth to look into it.

**Comment by slinlee** [comment]
Note: need to confirm why the requirements.txt is having issues

File: tests/examples/inventory_management/inventory_env_pathmind.py (hunk 1 above)
Comment (ejunprung) [reply tests/examples/inventory_management/inventory_env_pathmind.py:204]: @maxpumperla I think we got it working but we were confused about one thing. I'm using `Discrete(100, 3)` but it should be a tuple of various lengths instead. Like below.

Action 0: n = 100
Action 1: n = 90
Action 2: n = 80

Can simulation.py accommodate that?

**Comment by ejunprung** [comment]
@slinlee Is it ok if I merge? These should train fine in Pathmind. We can continue improving them later.

**Comment by ejunprung** [comment]
I'm going to merge this so that I can launch the tutorials. The test is failing because we removed the policy files since we want people to use policy server instead. Can fix that later.

**Pull request merged**